Description: remove upper bound in cryptography dependency
Author: Luca Boccassi <bluca@debian.org>
Forwarded: https://github.com/Azure/azure-cli/pull/15104
--- a/src/azure-cli/setup.py
+++ b/src/azure-cli/setup.py
@@ -129,7 +129,7 @@
     'azure-synapse-accesscontrol~=0.2.0',
     'azure-synapse-artifacts~=0.3.0',
     'azure-synapse-spark~=0.2.0',
-    'cryptography>=2.3.1,<3.0.0',
+    'cryptography>=2.3.1',
     'fabric~=2.4',
     'jsmin~=2.2.2',
     'pytz==2019.1',
